client/src/views/DashboardView.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-row class="ma-0 pl-4 pr-4 pt-4 pb-0" align="center">
+    <v-col>
+      <h1>Welcome back, {{ tempUserFirstName }}!</h1>
+      <!-- <v-divider/> -->
+    </v-col>
+  </v-row>
```

Comment:
This will center it horizontally
```
<v-row class="ma-0 pl-4 pr-4 pt-4 pb-0" align="center" justify="center">
    <h1>Welcome back, {{ tempUserFirstName }}!</h1>
  </v-row>
  ```